Abyss.Web/ClientApp/src/app/online/online.component.ts

```diff
@@ -26,6 +27,21 @@ export class OnlineComponent implements OnInit, OnDestroy {
                 this.clients = clients;
             });
         });
+        this.hub.onclose(() => {
```

Comment:
Also feels like the hub event subscriptions should live inside a `NgOnInit` hook rather than the constructor of the component

Reply:
I think that may end up with multiple subscriptions if you navigate away and back to the page as the hub object itself does not get destroyed